Fix musl build

Add missing header needed for ssize_t.

Signed-off-by: Baruch Siach <baruch@tkos.co.il>
---
Patch status: posted upstream
(http://lists.gnu.org/archive/html/bug-xorriso/2016-05/msg00000.html)

diff -Nuar xorriso-1.4.2-orig/libisofs/aaip_0_2.h xorriso-1.4.2/libisofs/aaip_0_2.h
--- xorriso-1.4.2-orig/libisofs/aaip_0_2.h	2015-11-28 16:41:33.000000000 +0200
+++ xorriso-1.4.2/libisofs/aaip_0_2.h	2016-05-24 11:40:43.874191174 +0300
@@ -16,6 +16,7 @@
 #ifndef Aaip_h_is_includeD
 #define Aaip_h_is_includeD yes
 
+#include <unistd.h>
 
 /* --------------------------------- Encoder ---------------------------- */
 
